### PR TITLE
[WIP] Started implementation of #94

### PR DESCRIPTION
**Pull request by jfversluis**
Introduced new property on the `IAmACommunityMember` interface. This needs to have a ISO 639-1 formatted language code.

Also implemented a unit test to cover this. The actual filtering is yet TBD

### CODE REVIEW

File: src/Firehose.Web/Web.config [hunk 1]
```diff
@@ -36,10 +36,6 @@
   <system.web>
     <compilation debug="true" targetFramework="4.5.1">
       <assemblies>
-        <add assembly="System.Globalization, Version=4.0.0.0,
```
Comment (Cheesebaron) [inline src/Firehose.Web/Web.config]: I don't think we should mess with this.

File: src/UnitTest/AuthorsTest.cs [hunk 2]
```diff
@@ -109,5 +109,24 @@ await _policy.ExecuteAsync(() => _httpClient.GetAsync(feedUrl))
...
+            var authorTypes = types.Where(t => typeof(IAmACommunityMember).IsAssignableFrom(t) &&
+                !_interfaceNames.Contains(t.Name)).ToArray();
+
+            foreach (var authorType in authorTypes)
+            {
+                var author = (IAmACommunityMember)Activator.CreateInstance(authorType);
+
+                Assert.False(string.IsNullOrWhiteSpace(author.FeedLanguageCode), $"{authorType.Name} has not specified a (valid) language code");
+                Assert.Equal(2, author.FeedLanguageCode.Length);
+                Assert.True(author.FeedLanguageCode.All(char.IsLower), $"{authorType.Name} specified language code is not lowercase");
```
Comment (Cheesebaron) [inline src/UnitTest/AuthorsTest.cs]: We should probably check against real C# `Culture` here.

File: src/Firehose.Web/Web.config (hunk 1 above)
Comment (jfversluis) [reply src/Firehose.Web/Web.config]: Whoops, VS4Mac decided this was a good idea on it's own. Will add it back.

File: src/UnitTest/AuthorsTest.cs (hunk 2 above)
Comment (jfversluis) [reply src/UnitTest/AuthorsTest.cs]: Ah, wow. Yeah, of course :)

**Comment by jfversluis** [comment]
Seems I can't approve since I did changes myself on this.. So going to use my magic override button here.

But awesome effort! Go us!

**Pull request merged**